core/internal/features_test.go

```diff
@@ -113,7 +113,7 @@ func TestIntegration_FeeBump(t *testing.T) {
 	config.Set("ETH_GAS_BUMP_THRESHOLD", 10)
 	config.Set("MIN_OUTGOING_CONFIRMATIONS", 20)
 
-	attempt1Hash := common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000001")
+	attempt1Hash := common.HexToHash("0xb7862c896a6ba2711bccc0410184e46d793ea83b3e05470f1d359ea276d16bb5")
```

Comment:
This was the line needed to make TestIntegration_FeeBump green, since we now set real tx hashes in the RunResult `result`. Even though we had the fake value `0x0000000000000000000000000000000000000000000000000000000000000001`, it was mismatching the real thing, preventing the job runner from finding the tx attempt.